packit/cli/init.py

```diff
@@ -46,58 +46,15 @@
 @click.option(
     "-f", "--force", is_flag=True, help="Reset config to default if already exists."
 )
-@click.option(
```

Comment:
since this is a backwards incompatible change, it could be frustrating to existing users who already tried this command

we should either have a warning here that the command has changed its interface or explicitly inform people about this: afaik only Mirek from virt used the command extensively (some other people tried it)

Reply:
Personally I'm not concerned that too many people are used to use `packit init` to initialize a source-git repo. Nevertheless, I've updated the help text for `packit init` to include a reference to `packit source-git init`. I'll also check documentation in packit.dev and update all the relevant parts. What do you think, would this be enough?

Reply:
See packit/packit.dev#290.